src/functional/ffront/field_operator_ast.py

```diff
@@ -165,7 +201,7 @@ class Compare(Expr):
 
 
 class Call(Expr):
-    func: Name
+    func: Expr
```

Comment:
Why is this an expression and not just a name?

Reply:
This was a temporary workaround for shifting temporaries:

```python
temp = inp_a + inp_b
return temp(Offset[value])
```

Gets inlined to `(inp_a + inp_b)(Offset[value])`. The longterm solution is to create a `Shift` node in FOAST.